src/pages/AdvancedSearch/CodeSnippetBlock/CodeSnippetBlock.tsx

```diff
@@ -139,6 +150,8 @@ const CodeSnippetBlock = ({ className, title, code }: Props): JSX.Element => {
     (valueAsString, value, ...keyPath) => {
       const currentKey = keyPath[0];
       const parentKey = keyPath.length >= 2 ? keyPath[1] : null;
+      const grandparentKey = keyPath.length >= 3 ? keyPath[2] : null;
```

Comment:
I definitely think there should be a better name for this, but couldn't find one. Should we just remove this line and input the value directly in line 209?